rero_ils/query.py

```diff
@@ -49,28 +49,48 @@ def and_search_factory(self, search, query_parser=None):
     :param search: Elastic search DSL search instance.
     :returns: Tuple with search instance and URL arguments.
     """
```

Comment:
Please add boosting support in the comment.

Reply:
done